addons/web/static/src/webclient/actions/action_service.js

```diff
@@ -1487,6 +1487,7 @@ export function makeActionManager(env, router = _router) {
                 ];
                 const context = {
                     ...action.context,
+                    ...(embeddedAction.context ? makeContext([embeddedAction.context]) : {}),
```

Comment:
not sure if makeContext is necessary here

Reply:
It is needed because the embeddedAction.context is a string. 🙂